### PR TITLE
Add blurb to exercises

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Closes https://github.com/exercism/v3-project-management/issues/159

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
@iHiD Could you check to see if the failing system tests are those known offenders?

**Comment by iHiD** [comment]
Yes, they look unrelated.

File: db/migrate/20210324081826_add_blurb_to_exercises.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class AddBlurbToExercises < ActiveRecord::Migration[6.1]
+  def change
+    add_column :exercises, :blurb, :string, null: true, limit: 350, after: :title
```
Comment (iHiD) [inline db/migrate/20210324081826_add_blurb_to_exercises.rb:3]: `after`. Posh!

**Comment by iHiD** [comment]
This is a great PR. Nice work :)

File: db/migrate/20210324081826_add_blurb_to_exercises.rb (hunk 1 above)
Comment (ErikSchierboom) [reply db/migrate/20210324081826_add_blurb_to_exercises.rb:3]: 🎉

**Pull request merged**